content_scripts/hud.coffee

```diff
@@ -114,6 +113,13 @@ HUD =
   unfocusIfFocused: ->
```

Comment:
Can we delete this `unfocusIfFocused` method and replace it with `giveUpFocus`? I find it confusing that there's now two methods with very similar names and functionality.

Reply:
Yes they're doing the same thing. I've updated the code.